wire/test/run-peer-wire.c

```diff
@@ -704,21 +703,22 @@ static bool channel_announcement_eq(const struct msg_channel_announcement *a,
 	return eq_upto(a, b, features)
 		&& eq_var(a, b, features)
 		&& eq_field(a, b, chain_hash)
-		&& structeq(&a->short_channel_id, &b->short_channel_id)
+		&& short_channel_id_eq(&a->short_channel_id,
+				       &b->short_channel_id)
 		&& eq_between(a, b, node_id_1, bitcoin_key_2);
 }
 
 static bool funding_locked_eq(const struct msg_funding_locked *a,
 			      const struct msg_funding_locked *b)
 {
-	return structeq(a, b);
+	return memcmp(a, b, sizeof(*a)) == 0;
```

Comment:
just curious why these aren't using structeq?

Reply:
Theory of minimal change, and laziness due to this being test code.  Otherwise, yes, we could have just STRUCTEQ_DEF() these into existence.